chart/values.yaml

```diff
@@ -660,6 +669,13 @@ config:
     enable_proxy_fix: 'True'
     expose_config: 'True'
     rbac: 'True'
+    authenticate: 'True'
+    auth_backend: 'airflow.contrib.auth.backends.google_auth'
+  google:
+    client_id: ''
+    client_secret: ''
+    oauth_callback_route: '/oauth2callback'  # this doesn't have any effect if rbac is turned on
```

Comment:
When we use `rbac` feature of airflow, this value doesn't have effect. The callback path (redirect path) is always set `oauth-authorized/<provider>`, that is, `oauth-authorized/google` so we need to register the path to "Authorized redirect URIs" of Google OAuth api.